packages/next-steps/style.module.css

```diff
@@ -170,11 +170,29 @@
 }
 
 .tileHeading {
+  display: flex;
+  align-items: center;
+  gap: var(--wpl-spacing-02);
```

Comment:
Wonder what the wrapping expectations are for longer titles. Should it stay placed to the right, or stay inline with the heading?

<img width="1706" alt="CleanShot 2022-12-17 at 06 52 10@2x" src="https://user-images.githubusercontent.com/825855/208240543-cc3a3905-ef19-4ffb-9295-a743ab799cd7.png">

Reply:
Good catch -  I totally missed this. The badge should wrap with the text. Updated [here](https://github.com/hashicorp/react-components/pull/851/commits/9a0393c85a2e3049bceeeffcdab2413970fdaf70).

Reply:
Is there an example in the codebase where we've done this? I might just be implementing in correctly but I'm not able to prevent an orphaned badge.

Reply:
Looks to be working as expected now! Not sure if I was testing the badge wrapping before the latest commits or not. Looks good!